lib/output.ts

```diff
@@ -35,7 +35,7 @@ export function mismatchingVersionsToOutput(
   return [
     `Found ${mismatchingDependencyVersions.length} ${
       mismatchingDependencyVersions.length === 1 ? 'dependency' : 'dependencies'
-    } with mismatching versions across the workspace.`,
+    } with mismatching versions across the workspace. Fix with \`--fix\`.`,
```

Comment:
Could chalk be used here instead of backticks?

Reply:
I don't immediately see a chalk formatting option that looks analogous to the backtick/code markdown formatting i.e. `--fix`.

Reply:
My idea would be to use bold formatting or something similar since the backticks will appear unformatted in a terminal.

Reply:
I could try that. One thing I want to be careful about however is that the output looks fine even for users with non-colored terminals.

Reply:
I tried bold and I don't think it made sense to highlight it like that.